### ISSUE TITLE
Documentar endpoints

### ISSUE THREAD

**Issue by paladini** [ranisalt/cagr-api#1]
Sei que você mal desenvolveu o projeto ainda, mas assim que possível seria muito legal documentar os endpoints da API. 

Parabéns pela ideia, realmente vai ajudar MUITO a fazer aplicações voltadas pra UFSC :)

**Comment by ranisalt** [comment]
Sim, na verdade eu to tentando reduzir o volume de banda do app ainda e vou documentar logo.
